schema/config.hcl

```diff
@@ -58,6 +58,12 @@ table "config_analysis" {
     null = true
     type = timestamptz
   }
+  column "is_pushed" {
```

Comment:
I think we might need to add a trigger on config_analysis to reset is_pushed when the row is updated (e.g. status or last_observed)